README.md

```diff
@@ -63,6 +63,52 @@ higher-level APIs:
 
 ## Guides
 
+### Quickstart
+
+1. Clone this repo:
+   ```
+   git clone https://github.com/kubernetes-sigs/kubefed.git
+   ```
+1. Start a kind cluster:
```

Comment:
suggest to add `kind` link for users to install `kind` before use it.

Reply:
Great idea! Added that.